Formula/deno.rb

```diff
@@ -0,0 +1,30 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno.git",
+      :tag      => "v0.2.8",
+      :revision => "a47d545e4d0831713d0107e86cead7b4506a9bf6"
+
+  bottle do
+  end
+
+  depends_on "ninja" => :build
+  depends_on "node@10" => :build
```

Comment:
The fact that it doesn't build with latest `node` is a blocker. Versioned formulas will be going away.

Reply:
I can remove that. I does build with latest node.